packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -402,33 +377,22 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
   });
 
   pgm.createTable(
-    "links",
+    "entity_relations",
     {
       source_entity_id: {
         type: "UUID",
         notNull: true,
         references: "entity_ids",
       },
-      target_entity_id: {
+      relationship_entity_id: {
```

Comment:
Not feeling strongly, but do we want to group `source_entity_id` and `target_entity_id` next to each other? Both approaches make sense.

Reply:
My thinking was that `source_entity_id` is related through `relationship_entity_id` to the `target_entity_id`. But may make more sense to group them :)

Reply:
Changed order in d397f935f2aff76455b959c2f2a16326c1496cdb :)